init-plugin.sh

```diff
@@ -26,8 +26,6 @@ prefix=$( perl -pe '$_ = lc; s/ /_/g' <<< "$name" )
 namespace=$( perl -pe 's/ //g' <<< "$name" )
 class=$( perl -pe 's/ /_/g' <<< "$name" )
 
-cd ..
```

Comment:
I guess if you're doing it the way you mentioned above like:
```
cd wp-content/plugins
./foo-bar/init-plugin.sh "My Awesome Plugin"
```

However, I was doing it like the direction in the readme like so
```
cd wp-content/plugins/foo-bar
./init-plugin.sh "My Awesome Plugin"
```

If you do it this way the directory is created inside the `foo-bar` directory which is kind of weird. I suggest we do a check for the current working directory and if it's `foo-bar` then we `cd ..` back up to the plugins directory to create the new plugin. That way both use cases are handled and it does create plugins inside plugins. :smile: